### PR TITLE
Add Warren Chan to Contributors

### PR DESCRIPTION
**Pull request by warrenchan13**
_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._

## Description
In this PR, Warren is added to CONTRIBUTORS.md 

## Acceptance Criteria
Warren is in the file with the link to the GitHub repo + favorite emoji

## Type of Changes
`enhancement`

### Before
Warren was not in the CONTRIBUTORS file.

### After
Warren is in the CONTRIBUTORS file.

## Testing Steps / QA Criteria
Verify that Warren appears in the CONTRIBUTORS file.

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ Add yourself as a contributor to this project as a Markdown link that links your
 ## Cohort Participants
 
 Gabrielle Lommel (https://github.com/granolagabrielle) :seedling:
+Warren Chan (https://github.com/warrenchan13) 💪
```
Comment (ahsanatzapier) [inline CONTRIBUTORS.md:10]: Hey, @warrenchan13! Almost there! Let's use the same syntax in the example above to get the markdown formatting:

[name](link) emoji

Could you also update @granolagabrielle entry? Thanks!

Comment (warrenchan13) [reply CONTRIBUTORS.md]: Updated mine and @granolagabrielle's entry

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -6,8 +6,8 @@ Add yourself as a contributor to this project as a Markdown link that links your
 
 ## Cohort Participants
 
-Gabrielle Lommel (https://github.com/granolagabrielle) :seedling:
-Warren Chan (https://github.com/warrenchan13) 💪
+- [Gabrielle Lommel](https://github.com/granolagabrielle) :seedling:
+- [Warren Chan](https://github.com/warrenchan13) 💪
```
Comment (ahsanatzapier) [inline CONTRIBUTORS.md:10]: Thanks for updating this! 🚀

**Review by ahsanatzapier — APPROVED** [review]
Looks great! And thanks for updating the existing entry! 🚀

**Pull request merged**